Add OEO to ENERO Foundry ontologies (#68)
ENEROFoundry/ENEROFoundry: docs/ontologies.md

# Ontologies

The ENERO Foundry Assembly evaluate energy-related ontologies for compliance with the [ENERO Principles](principles/).

> Each evaluation reviews an ontology against all ten principles and records whether each one is

Reviewer: ```suggestion
Each evaluation reviews an ontology against the principles and records whether each one is
```

> fully met, partially met, or not yet implemented.

Reviewer: ```suggestion
fully met, partially met, not yet implemented, or not yet evaluated.
```
